sage_modabvar/abvar.py

```diff
@@ -2171,10 +2177,9 @@ def number_of_rational_points(self):
         else:
             return self.rational_torsion_subgroup().order()
 
-    def frobenius_polynomial(self, p):
+    def frobenius_polynomial(self, p, var='x'):
```

Comment:
Added the var option so it was consistent with other polynomial functions like charpoly and minpoly.